bigquery_etl/cli/query.py

```diff
@@ -1526,6 +1526,10 @@ def deploy(
             click.echo(f"{query_file} dry runs are skipped. Cannot validate schemas.")
             continue
 
+        if str(query_file).endswith(".py"):
+            click.echo(f" Cannot validate schemas for python file {query_file}")
+            continue
+
```

Comment:
We'd still like to be able to deploy `schema.yaml` files for python scripts, but skip the validation part for them. I think this would skip the schema deploy for python files in general.
We'll probably need to add a check around the `if not force:` line to see if a python script has a `schema.yaml` file that could be deployed